src/data/SpectraManager.ts

```diff
@@ -123,3 +130,56 @@ export function toJSON(
     });
   }
 }
+
+export function exportAsJcamp(spectrum: Datum1D | Datum2D) {
+  let jcamp: string | null = null;
+  if (Spectrum1D.isSpectrum1D(spectrum)) {
+    jcamp = toJcamp1D(spectrum);
+  } else {
+    throw new Error('convert 2D spectrum to JCAMP is not supported');
+  }
+
+  const blob = new Blob([jcamp], { type: 'text/plain' });
+  saveAs(blob, `${spectrum.info.title}.jdx`);
+}
+
+function toJcamp1D(spectrum: Datum1D) {
```

Comment:
Please move this code to nmr-load-save. It will also be easier to add a testcase.

The dependency convert-to-jcamp should also be only in nmr-load-save.

You may expose the new method `spectrum1DToJcamp`